generation/internal/php.go

```diff
@@ -34,6 +36,16 @@ func initialPhpSteps(ls labels.LabelSet) []config.Step {
 	return []config.Step{checkout, installPackages}
 }
 
+func phpImageVersion(composerVersion string) string {
+	version := "8.2.7"
```

Comment:
~ We have been defining versions/docker image tags as package level constants, mostly.

(Thinking out loud, for a future PR) Should we move them all to a `versions.go` file or such? Maybe we can even write some test to check if they are up-to-date?